src/cp/cp_api.c

```diff
@@ -233,7 +234,7 @@ __wt_conn_control_point_enable(WT_CONNECTION *wt_conn, wt_control_point_id_t id,
         return (WT_ERROR);
     cp_registry = &(conn->control_points[id]);
     cfgs[0] = conn->cfg;
```

Comment:
Since you have renamed the function parameter to extra_cfg, rename cfgs[] to cfg[].